packages/@apphosting/adapter-angular/src/utils.ts

```diff
@@ -142,6 +143,16 @@ function extractManifestOutput(output: string): string {
   return stripAnsi(output.substring(start, end + 1));
 }
 
+export function populateMetadata(angularVersion?: string): Metadata {
```

Comment:
tsdocs for exported functions please

go/tsjs-style#document-all-top-level-exports-of-modules

Think of exported functions as your file/module/class's API. Unless it's exceedingly obvious what a function does (eg a basic getter), doc strings are usually a safe bet